package.json

```diff
@@ -109,8 +112,9 @@
     "npm-shrinkwrap.json"
   ],
   "scripts": {
-    "test": "NODE_ENV=test jest",
-    "lint": "eslint Examples/ Libraries/",
+    "test": "NODE_ENV=test node_modules/jest-cli/bin/jest.js",
```

Comment:
this shouldn't be necessary? `npm install` creates a `.bin` folder in `node_modules` with a `jest` executable. `npm run` adds this to the $PATH.